spec/models/institution_builder_spec.rb

```diff
@@ -891,6 +891,17 @@
       end
     end
 
+    describe 'when generating institution programs' do
+      let(:institution) { institutions.find_by(zip: zipcode_rate.zip_code) }
+      let(:institution_program) { InstitutionProgram.first }
+
+      it 'properly generates institution programs from weams data' do
+        create :program
+        InstitutionBuilder.run(user)
+        expect(institution_program).not_to eq(nil)
```

Comment:
It's unclear to me that `InstitutionBuilder.run` caused `institution_program` to not be `nil`.
Perhaps we can more clearly test that `InstitutionBuilder.run` is causing `InstitutionProgram` to be created.
Something like:
```
     it 'properly generates institution programs from weams data' do
        create :program
        expect { InstitutionBuilder.run(user) }.to change{InstitutionProgram.count}.from(0).to(1)
      end
```